### PR TITLE
Feature selection example with scikit-learn and py operator

### PR DESCRIPTION
**Pull request by chezou**
This PR introduce feature selection with scikit-learn for boston house pricing. 

- Introduce regression-py.dig
- Add hyper parameter tuning workflow(search_hyperparams.dig) as well

### CODE REVIEW

**Comment by myui** [comment]
`Pipfile.lock` should be removed.

File: machine-learning/house_price/regression-py.dig [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    +test:
+      td>: queries/split_test.sql
+      engine: presto
+      create_table: ${source}_test
+
+  +feature_selection:
+    py>: tasks.FeatureSelector.run
+    docker:
+      #image: 'python:3.6.5'
+      image: 'chezou/td-ml-base:latest'
```
[the remark this answers is not in the document]
Comment (muga) [reply machine-learning/house_price/regression-py.dig]: Good point. Please comment it in the source code directly.

**Comment by chezou** [comment]
`Pipfile.lock` is for testing local python environment. The purpose for it is similar to `Gemfile.lock` to pin specific version of packages. If we remove `Pipfile.lock`, we have to remove `Pipfile` as well. Or, We have to keep both files.

File: machine-learning/house_price/regression-py.dig (hunk 1 above)
[the remark this answers is not in the document]
Comment (chezou) [reply machine-learning/house_price/regression-py.dig]: Noted 68e65e7

**Comment by myui** [comment]
master branch is for available feature in production. Better to note `>py` is WIP feature.

**Comment by chezou** [comment]
OK, noted 6cfd0ed

**Comment by myui** [comment]
better to add readme.md that notices wip/unstable feature.

Users might not see comments.

**Comment by chezou** [comment]
This workflow is an option for existing house pricing prediction. I've already commented on the README.md for the description of this workflow https://github.com/treasure-data/workflow-examples/commit/6cfd0ed655650c576a06478b996b6c5e15720755 . If there is a better place to mention it, I would be happy if you could tell me.

File: machine-learning/house_price/regression.dig [hunk 2]
```diff
@@ -32,13 +31,13 @@ _export:
 #     eta0: [5.0, 1.0, 0.5, 0.1, 0.05, 0.01, 0.001]
 
 #   _parallel: true
-  
+
```
Comment (myui) [inline machine-learning/house_price/regression.dig]: better to remove unused codes and working codes. 

Keep to it simple.

Comment (chezou) [reply machine-learning/house_price/regression.dig]: Nice catch! Removed 7901591

**Pull request merged**